docs/sources/plugins/developing/dataframe.md

```diff
@@ -45,15 +45,15 @@ With dataframes, each column is represented by a **Field**. So the essence of a
 
 One restriction on dataframes is that all Fields in the frame must be of the same length to be a valid dataframe.
 
-### When is a Dataframe a Time Series or a Table
+### When is a dataframe a time series or a table
```

Comment:
Are "time series" and "table" proper nouns? Are they the names of something? I ask because you capitalize them earlier, but you lowercase them here. Please make them consistent.

Reply:
Maybe. I don't really know... Dataframe could be considered a name I guess. Timeseries/table are formats.. Im not sure about this one. @kylebrandt  !!!

Reply:
@oddlittlebird do we by any chance have a list of Grafana specific words and how we spell/capitalize them?

Reply:
Yes, https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#capitalization.

However, these are sort of a gray area. If a word is used generically, then it should not be capitalized. This looks like the case here.

Reply:
Agree. Let's go lowercase